### DISCUSSION TITLE
Programatially install docsets

### DISCUSSION THREAD

**Discussion by ttanay** [zealdocs/zeal#1535] [Q&A] [answered]
I use Zeal heavily to look at docs while working. I'm trying to make my dev environment reproducible. 
To that end, I'd like to programmatically specify docsets I want to install with each installation of zeal.
Is there an existing way to do this?

**Comment by trollixx** [comment, marked as answer]
There's no way to do this right now. See #509 and #1368. You can copy the whole docset folder. I think someone also created a Python script which could download docsets and make them available to Zeal.

**Reply by ttanay** [reply]
This is really helpful! Thanks @trollixx!
